src/Services/ValidationService.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace ITBugTracking\Services;
+
+use ITBugTracking\Hydrators\SeverityHydrator;
+use PDO;
+
+class ValidationService
+{
+
+    public static function checkRequiredDataExists($data): bool
+    {
+        return !empty($data['reporter']) || !empty($data['title']) || !empty($data['severity']) || !empty($data['department']);
+    }
+
+
+    //sanitize 255 character limits
+    public static function limitTitleCharLengthTo255($data)
+    {
+        if (isset($data)) {
+
+            if (strlen($data) > 255) {
+                $data = substr($data, 0, 255);
+            }
+            return $data;
+        }
+    }
+
+    public static function limitReporterCharLengthTo255($data)
```

Comment:
```suggestion
    public static function limitReporterLength($data)
```

Don't include specific figures in function names, because figures are easy to change and function names are more annoying to change.

Reply:
done